packages/release-mastermind/types/index.d.ts

```diff
@@ -3,0 +1,266 @@
+import {
+  Condition,
+  IssueCondition,
+  PRCondition,
+  ProjectCondition,
+  ScheduleCondition
+} from '../src/conditions'
+import { Repo } from '../src/utils'
+
+/**
+ * Application interfaces
+ */
+
+export interface Options {
+  configPath: string
+  configJSON: Runners
+  showLogs: boolean
+  dryRun: boolean
+  fillEmpty: boolean
+  skipDelete: boolean
+  repo?: Repo
+}
+
+export interface Runners {
+  labels?: Labels
+  runners: Config[]
+}
+
+export interface Config {
+  root: string
+  versioning: {
+    source: VersionSource
+    type?: VersionType
+  }
+  retryLimit?: number
+  prereleaseName?: string
+  labels?: { [key: string]: string }
+  sharedConfig?: SharedConfig
+  pr?: PullRequestConfig
+  issue?: IssueConfig
+  project?: ProjectConfig
+  schedule?: ScheduleConfig
+}
+
+/**
+ * Config types
```

Comment:
Codacy has a fix for the issue: [Sentence must end with a period.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
 * Config types.
```